File: openapi/code_samples/python/api_v1_roles/post.py
from scalekit.v1.roles.roles_pb2 import CreateRole

role = CreateRole(
    name="admin",
    display_name="Admin",
    description="Environment-level role",
    extends="base_role",                  # optional
    permissions=["read:users"]           # optional
)

scalekit_client.roles.create_role(role=role)

File: openapi/code_samples/go/api_v1_roles/post.go
resp, err := scalekitClient.Role().CreateRole(ctx, &rolesv1.CreateRole{

	Name:        "admin",

	DisplayName: "Admin",

	Description: "Environment-level role",

	Extends:     proto.String("base_role"),      // optional

	Permissions: []string{"read:users"},        // optional

})